src/frontend/src/icp/services/icrc.services.ts

```diff
@@ -152,6 +152,7 @@ const loadCustomIcrcTokensData = async ({
 
 		const indexCanisterId = fromNullable(index_id);
 
+		// TODO(OISY-296): remove isNullish(indexCanisterId) when support for reading balance and no index is fully implemented
```

Comment:
Just to double-confirm: you moved this TODO to the other file, right?

Reply:
What what? which other file? moved? That's a new TODO.

Reply:
Sorry, forget it: I thought I saw another `TODO(OISY-296)` with the same definition being removed somewhere, but I cannot find it anymore, I must have been mistaken

All good!